wc-admin.php

```diff
@@ -192,3 +192,8 @@ function wc_admin_woocommerce_updated() {
 	WC_Admin_Notes_Settings_Notes::add_notes_for_settings_that_have_moved();
 }
 add_action( 'woocommerce_updated', 'wc_admin_woocommerce_updated' );
+
+/*
+ * Remove the emoji script as it always defaults to replacing emojis with Twemoji images.
```

Comment:
Should we link to the related gutenberg url you mentioned in your PR description here?

Reply:
definitely, fixed.